temporian/core/evaluation.py

```diff
@@ -258,6 +258,59 @@ def build_schedule(
     return schedule
 
 
+def has_leak(
+    output: EvaluationQuery,
```

Comment:
EvaluationQuery seems to be creeping into the codebase, added TODO in my last PR to give it a better name

Reply:
sg